packages/smtml/smtml.0.1.2/opam

```diff
@@ -0,0 +1,50 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A Front-end library for SMT solvers in OCaml"
+description: "A Multi Back-end Front-end for SMT Solvers in OCaml."
+maintainer: ["Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>"]
+authors: ["Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>"]
+license: "GPL-3.0-only"
+homepage: "https://github.com/formalsec/smtml"
+doc: "https://formalsec.github.io/smtml/smtml/index.html"
+bug-reports: "https://github.com/formalsec/smtml/issues"
+depends: [
+  "dune" {>= "3.14"}
+  "ocaml" {>= "4.14.0"}
+  "ocaml_intrinsics" {< "v0.17.0"}
```

Comment:
```suggestion
```

will be fixed in https://github.com/ocaml/opam-repository/pull/26133

Reply:
that's merged, so this should be good to go

Reply:
This actually deleted the dependency instead of just removing the constraint. But I'll add it back in :sweat_smile: